app/controllers/comments_controller.rb

```diff
@@ -1,40 +1,35 @@
 class CommentsController < ApplicationController
   before_filter :authenticate_user!
+  before_filter ->{authorize self.proposal, :can_show!}
```

Comment:
`can_show!` is a policy on proposals, not comments. So, to create/view/etc. comments, we are testing the ability to see the associated proposal.

We could add a new method which performs the same action (`can_create_comment`?) but that seems to obscure the test.